frontend/packages/dev-console/src/components/import/AdvancedOptions.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { filter } from 'lodash';
+import { scrollToBottom } from '../../utils/scroll-utils';
+import './AdvancedOptions.scss';
+
+type AdvancedOptionProps = {
+  options: {
```

Comment:
Maybe if you make `options` as array of objects i think you can simplify the rendering process and you wouldn't have to save `optionsOrder` separately.

Reply:
@rohitkrai03 `optionsOrder` is there to maintain the order in which items are clicked. If I make it an array of objects still I would have to reshuffle the array or create a new array push that object in it for which an item clicked. That's why I kept `options` an object so that I wouldn't have to search for the clicked item in an array.